default/templates/default/base-uploader.html

```diff
@@ -12,8 +12,23 @@
 
 {% block body %}
 <div class="response-success alert alert-info hidden">
-    <b>{% trans "Success!" %}</b> result.zip(<span class="file-size"></span>)
-    <a href="javascript:void(0)" class="download" target="_blank">{% trans "Download" %}</a>
+    <b>{% trans "Success!" %}</b>
+    result.zip(<span class="f-file-size"></span>)
+    <a class="btn btn-primary btn-download f-file" href="#" role="button">
+        {% trans "Download" %}
+    </a>
+    <span>  /  </span>
+    <button type="button" class="btn btn-primary btn-download d-drive">
+        <span>{% trans "Save to Google Drive" %}</span>
+    </button>
+</div>
+<div class="google-drive-success alert alert-info hidden">
+    <b>{% trans "Uploaded to Google Drive!" %}</b>
+    <ul>
+        <li>
+            <a class="btn btn-primary btn-download file-google" target="_blank" href="#" role="button"></a>
```

Comment:
Me imagino que aquí se inserta el nombre del archivo, ¿no? Si esto luce como un botón sería mejor que tenga un texto genérico como "Open in Google Drive" o algo parecido.

Reply:
Ahora mismo se ve así:
![imagen](https://user-images.githubusercontent.com/48607824/80733836-3b446e00-8adc-11ea-9de9-065c34f446f3.png)

Lo podría cambiar por:
![imagen](https://user-images.githubusercontent.com/48607824/80734289-f0772600-8adc-11ea-897b-9d106b931bff.png)